### ISSUE TITLE
Feature: Click to expand folder

### ISSUE THREAD

**Issue by that0n3guy** [aidenlx/alx-folder-note#32]
It would be nice if there was an option to click to expand the folder.   Right now, if I click on the folder it opens the folder note, I then after to click the arrow ![image](https://user-images.githubusercontent.com/113870/141538464-9d50860d-26b3-4413-a4f1-a0aa67a3d8b2.png) to expand the folder and see whats in it.

Can you make an option that when the folder is clicked on the folder expands AND the note is displayed?

**Comment by Peeeet** [comment]
Unless I'm missing something, the setting for this doesn't seem to appear in the settings pane.  I can't tell from the code, is it supposed to show up in the settings pane for the "Folder Note Core" Plugin or the "AidenLX's Folder Note" plugin?  Because the Core plugin doesn't seem to have a settings page...

**Comment by aidenlx** [comment]
@Peeeet it’s not published yet, and this issue will be closed automatically once the version is published.

**Comment by Peeeet** [comment]
Ah okay I only commented because something in a recent update changed how clicking on folders worked:

- It used to be that I could click the text of the folder name in the explorer to open the folder note, and click on the empty space to the right of the text in the explorer to expand / collapse said folder.
- Now instead, clicking in either area just opens the folder note.

This change did end up preventing me from expanding / collapsing the folders in the explorer, but only because I had a CSS snippet that hid the little arrow to the left of the folder name text in the explorer view.  I realize that the newly described behavior is probably more "expected", but since I was using said CSS snippet, the "wrong" way was how I had been working for a while.  I commented here because I had assumed the change of functionality was related to this implementation, but I guess not :-) so I'll just wait till I see the option appear to try it out and see if I can go back to my old CSS snippet.

Regardless, thanks for keeping folder-note alive!
